docker-compose.dev.yml

```diff
@@ -29,6 +29,7 @@ services:
   prometheus:
     image: prom/prometheus:latest
     restart: always
+    user: "${UID}:${GID}"
```

Comment:
this is to fix that weird issue where Prometheus cannot access its necessary directories

Reply:
would this work on windows?

Reply:
since `UID` and `GID` are not set on windows, docker may ignore the line entirely
requires further testing